### PR TITLE
Improve graphics fill warnings and button to remove graphic

### LINKED ISSUE #3444: Fill symbolizer with graphic fill ignores fill color
# Bug

**Describe the bug**
In Styler, Fill symbolizer allows to specify a graphic fill (markers or icons, as seen on the screenshot) and a fill color at the same time. When I tried that, it seems that the symbolizer ignores the fill color completely and uses some default fill value instead.

**To Reproduce**
1. Go to 'Styler'
2. Click on 'Fill Symbolizer'
3. Click on '+' near the "Graphic Fill"
4. See the fill color change

**Expected behavior**
The fill color is preserved.

**Screenshots**
image

**Additional context**
none

### PR DESCRIPTION
**Pull request by raitisbe**
## Description

Warn user about fill color being ignored when graphic fill is present

## Related issues or pull requests

fix #3444

Please list issues or pull requests that the changes you propose are related to. It does not matter if they are still open and/or unmerged, any link is appreciated.

## Pull request type

Please check the type of change your PR introduces:

- [ ] Bugfix
- [x] Feature
- [ ] Dependency updates
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe)
- [ ] I am unsure (we'll look into it together)

## Do you introduce a breaking change?

- [ ] Yes
- [x] No
- [ ] I am unsure (no worries, we'll find out)

## Checklist

- [x] I understand and agree that the changes in this PR will be licensed under the [MIT License]
- [x] I have followed the [guidelines for contributing](https://github.com/hslayers/hslayers-ng/blob/master/CONTRIBUTING.md)
- [x] The proposed change fits to the content of the [code of conduct](https://github.com/hslayers/hslayers-ng/blob/master/CODE_OF_CONDUCT.md)
- [ ] I have added or updated tests and documentation, and the test suite passes (run `npm test` locally)
- [ ] I'm lost; why do I have to check so many boxes? Please help!

### CODE REVIEW

**Comment by jmacura** [comment]
Thanks for this change. It makes the consequences a way cleaner. I have re-worded the strings a bit. Would like @FilipLeitner to check the SK translations.
Additionally I have added a change that displays the Mark and Icon symbolizers in a bit different style when they are used as part of the graphic symbolizer (not as a symbolizer on its own). See below.
image

File: projects/hslayers/src/components/styles/symbolizers/fill-symbolizer/fill-symbolizer.component.html [hunk 1]
```diff
@@ -21,23 +24,32 @@
...
       [title]="'STYLER.warningAttribStoredNotVisualized' | translateHs : {app} ">!</span>
     <div class="p-1">
-      <div ngbDropdown #graphicFillMenu="ngbDropdown" class="d-inline-block" placement="top-right">
-        <button class="btn btn-outline-primary btn-sm" id="dropdownAddFillSymbolizer" ngbDropdownToggle>
-          <span class="icon-plus"></span>
-        </button>
-        <div ngbDropdownMenu aria-labelledby="dropdownAddFillSymbolizer">
-          <button ngbDropdownItem (click)="addSymbolizer('graphicFill', 'Mark')">Mark</button>
-          <button ngbDropdownItem (click)="addSymbolizer('graphicFill', 'Icon')">Icon</button>
+      <div class="btn-group me-3">
```
Comment (jmacura) [inline projects/hslayers/src/components/styles/symbolizers/fill-symbolizer/fill-symbolizer.component.html:27]: See #3648 is it really the cause for `me-3`?

**Review by jmacura — APPROVED** [review]
The changes are fine with me.
One question remains though: Why it is possible to set a "graphic fill" on a "line symbolizer"? I can not imagine how that should be visualised 😄 ("graphic stroke" I can imagine, on the other hand)

File: projects/hslayers/src/components/styles/symbolizers/fill-symbolizer/fill-symbolizer.component.html (hunk 1 above)
Comment (raitisbe) [reply projects/hslayers/src/components/styles/symbolizers/fill-symbolizer/fill-symbolizer.component.html:27]: > See #3648 is it really the cause for `me-3`?

No no, that line was only tp get the two buttons together. me-3 might even not be necessary, I just copied it from somewhere

**Comment by raitisbe** [comment]
> 



> Why it is possible to set a "graphic fill" on a "line symbolizer"?

It's supported by geostyler. See: https://geostyler.github.io/geostyler-style/docs/master/interfaces/LineSymbolizer.html . It says that graphicFill is used with PointSymbolizer to "Renders the pixels of the line with a repeated pattern., where graphicStroke "Renders the line with a repeated linear PointSymbolizer.". Can try and see the difference

**Pull request merged**